models/user.go

```diff
@@ -945,13 +945,23 @@ func deleteUser(e *xorm.Session, u *User) error {
 	// ***** END: Star *****
 
 	// ***** START: Follow *****
+	followees := make([]*Follow, 0, 10)
```

Comment:
Maybe this could be changed to []int64 array and select only FollowID values into this array. I don't know what is the limitation of count for `In(field, array)` but it could be optimized to update it in batches so that there would be less requests to database